components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -581,6 +581,11 @@ - (BOOL)needsUpdatePlaceholderAnimationConstraintsToUp {
 // itself. That means these constraints need go away. So, we can use the existence of them as a
 // way to check if the placeholder is floating up. See isplaceholderUp.
 - (void)cleanupPlaceholderAnimationConstraints {
+  if (self.placeholderAnimationConstraintLeading.active &&
```

Comment:
Would you please add a small comment explaining why we would exit early under these conditions?

Reply:
Did you mean to add a comment to the code?

Reply:
I can add to code something like this: "We need to clear only deactivated constraints and to prevent the clearing of active constraints."
It's OK?

Reply:
Yep, that sounds good. Especially the second part. 👍